data/scripts/lib/register_monster_type.lua

```diff
@@ -225,6 +232,10 @@ registerMonsterType.loot = function(mtype, mask)
 					end
 					if children.subType or children.charges then
 						child:setSubType(children.subType or children.charges)
+					else
+						if charges > 0 then
+							child:setSubType(charges)
```

Comment:
is it normal to place the default charges from parent to child? shouldn't I get the charges of the itemtype of the child id in question?

Reply:
nah, should make a new variable from `ItemType(child.id):getCharges()`

Reply:
correct now?

Reply:
haven't tested, but looks ok

Reply:
![Untitled](https://user-images.githubusercontent.com/43324808/161484583-6776f80f-9238-4024-a7b0-d7f38ed02851.png)